robber/matchers/contain.py

```diff
@@ -5,16 +5,33 @@
 
 class Contain(Base):
     """
-    expect({'key': value}).to.contain('key')
-    expect([1, 2, 3]).to.contain(2)
+    expect({'key': value}).to.contain('key 1', 'key 2', 'key n')
+    expect([1, 2, 3]).to.contain(1, 2, 3)
     """
 
     def matches(self):
-        return self.expected in self.actual
+        expected_list = list(self.args)
+        expected_list.insert(0, self.expected)
+
+        if not self.is_negative:
+            for expected in expected_list:
+                if expected not in self.actual:
+                    self.expected_arg = expected
+                    return False
+
+            return True
+        else:
+            # As this is the negative case, we have to flip the return value.
+            for expected in expected_list:
```

Comment:
A contains B == A contains all elements of B (1) (B1 in A and B2 in A and ... Bn in A)
A not contains B == A not contains all elements of B (2) (B1 not in A and B2 not in A and ... Bn not in A)
not (1) != (2)

I cannot find another better way to explain, hope you understand :sweat_smile: